hackathon_site/event/jinja2/event/landing.html

```diff
@@ -226,26 +226,16 @@ <h3 class="center white-text">FAQ</h3>
 <div class="ombreBackgroundDark2">
     <div class="container section scrollspy" id="sponsors">
         <div class="row">
-<!--            <h3 class="center white-text">Sponsors</h3>-->
+            <h3 class="center white-text">Sponsors</h3>
             <div class="row sponsorImgs">
                 <!-- Platinum -->
-<!--                <a href="https://www.daisyintelligence.com/" class="col m6 s12" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Daisy_Intelligence_white.svg') }}" alt="Daisy Intelligence"/></a>-->
-<!--                <a href="https://tractionondemand.com/" class="col m6 s12" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Traction_On_Demand_white.svg') }}" alt="Traction on Demand" /></a>-->
-
-                <!-- Gold  offset-m2 offset-s1-->
-<!--                <a href="https://www.mathworks.com/" class="col m4 s5" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Mathworks_white.svg') }}" alt="Mathworks" /></a>-->
```

Comment:
Fyi, the class "col m4 s5" means that that the image is a column and when the page width is medium (m), the image should take up 4 columns in a 12 column layout, and when the page width is small (s), the image should take up 5 columns in a 12 column layout. 

When I first designed this section, I made sure the m6 s12 makes sense to platinum, m4 s5 for Gold, etc. They get smaller the lower the sponsorship tier. I also manually added offsets just to make sure things were spaced out evenly. Adding offsets is based off the number of sponsors you have each year. If you're going to delete all this, make sure that you remember the class for each tier. Or you can leave them commented out and refer to them as you add logos.

Read more about it here: https://materializecss.com/grid.html

Reply:
Okay this makes sense, I'll revert to the original format

Reply:
Ah actually on second thought I think the number of columns we use are going to change depending on how many sponsors we get for each tier, so I'm just going to leave a comment saying we should decrease logo size based on tier